### PR TITLE
WIP: Create clammit av service

### PR DESCRIPTION
**Pull request by bseeger**
Resolves https://www.pivotaltracker.com/story/show/185770088

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2220,6 +2220,15 @@ form-flow:
...
 | `form-flow.uploads.thumbnail-height`    | `60`                                                       | Thumbnail height in pixels                                                                                                            |
 
+##### Virus scanner properties
+
+| Property                                                | Default | Description                                                                                                                                        |
+|---------------------------------------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `form-flow.uploads.virus-scanning.enabled`              | `true`  | Turns on virus scanning component                                                                                                                  |
+| `form-flow.uploads.virus-scanning.clammit-url`          | None    | Full path for scanning on hosted (clamav-server)                                                                                                   |
```
Comment (bseeger) [inline README.md:2235]: I'm not sure we want to call this `clammit-url` - maybe just `service-url`. Do we want this to be generic to be used by any implementation of a scanning service?

Comment (bseeger) [reply README.md:2235]: (I realize this is a sw arch question and not just a doc thing)

File: src/test/java/formflow/library/file/UploadJourneyTests.java [hunk 2]
```diff
@@ -58,6 +61,7 @@ void documentUploadFlow() {
     } catch (InterruptedException e) {
       throw new RuntimeException(e);
     }
+    takeSnapShot("test.png");
```
Comment (bseeger) [inline src/test/java/formflow/library/file/UploadJourneyTests.java:64]: Do we need this here beyond this ticket?   (I realize this isn't ready for review yet, just wondering)

File: README.md (hunk 1 above)
Comment (bencalegari) [reply README.md]: Good idea, I'll switch to `service-url`

File: src/test/java/formflow/library/file/UploadJourneyTests.java (hunk 2 above)
Comment (bencalegari) [reply src/test/java/formflow/library/file/UploadJourneyTests.java:64]: Nope, accidentally left that in.

**Pull request merged**